src/components/NavBar/NavBar.scss

```diff
@@ -234,11 +241,11 @@
       top: 54px;
       left: 0;
       z-index: 100;
-      padding: 0 10px;
+      padding: 0 12px;
       position: fixed;
       width: 100%;
       height: 100vh;
-      background: white;
+      background: #2c73d6;

```

Comment:
@shreyas-coditas Please check if we have variable for this color. If we don't and color is used at multiple places, please create new variable.